README.md

```diff
@@ -90,14 +92,11 @@ git clone --recursive https://github.com/shmuelzon/esp32-ble2mqtt
 
 Modify the [configuration file](#configuration) to fit your environment, build
 and flash (make sure to modify the serial device your ESP32 is connected to):
+
 ```bash
 idf.py flash -p /dev/ttyS1
```

Comment:
Since we've removed all the references to the serial device, please also remove it from the command here.
I suggest to change it to `idf.py build flash` so it's a bit more verbose.

Reply:
Sure thing, updated!